src/parse.js

```diff
@@ -102,17 +103,24 @@ export default function parse (str, {meta} = {}) {
 							Object.assign(meta, {formatId: "color", types});
 						}
 
+						if (colorSpec.id.startsWith("--") && !id.startsWith("--")) {
```

Comment:
I think the warning messages could be made more clear

`Use color(--hsluv ...) instead of color(hsluv ...)`

Not super happy with this suggestion so feel free to ignore it.

Reply:
Actually, I'm wondering if we really need the warning messages. It's not clear to me who the warning messages are for. Developers? End Users?

As an end user of the convert app I'm happy that the app is liberal in what it accepts as I frequently forget to use the \<dashed-ident\> so I don't care if there's a warning message in the console.

Running the tests in the browser prints out a bunch of warning messages and I'm assuming that running the tests from the command line will do the same.

Reply:
Well, the prefixing *does* serve a purpose. That said, I’m happy to not have warnings for now, and become stricter later.

Reply:
As a developer I personally like the warnings because they flag an issue without preventing me from using it, but I can remove these if you like -- @LeaVerou is that your preference?

Reply:
What about disabling the warnings if we're in a test environment?

The htest cli runner could set `process.env.NODE_ENV` to `"test"`. Not sure if there is a convention is for browser test runners.

Reply:
What @jgerigmeyer meant was that the warnings are helpful for developers using Color.js, not just for us running tests. I think it would be good to have a production version and a development version, like Vue does, though that's a much bigger change. I vote to keep the warnings for now.

Reply:
My concern was more about seeing the warnings every time I run the conversion tests. But it's not a showstopper.

Reply:
That makes some sense to me @lloydk. I pushed one approach that skips these warnings in tests, @LeaVerou what do you think? #441